api/src/main/java/jakarta/persistence/NamedQueryReference.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2008, 2023 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Should it really be a 2008 copyright?

Reply:
Ah, I guess not, I just copy/pasted.

Reply:
Fixed, and added missing `Contributors`

Reply:
I'm fixing these in one go later, see ie #480 or #497 (which also added .git-blame-ignore-revs)